File: Data Structures & Algorithms/duplicate-integer/submission-4.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        if (nums.empty()) return false;
        sort(nums.begin(), nums.end());
        for(int i = 0; i < (int)nums.size() - 1; i++){
            int j = i + 1;
            if((nums[i] - nums[j]) == 0){
                return true;
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/is-anagram/submission-4.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False

        t_list = list(t)
        s_list = list(s)
        
        t_list.sort()
        s_list.sort()

        return all(t_list[i] == s_list[i] for i in range(len(t_list)))